docs/cli.rst

```diff
@@ -56,6 +56,18 @@ If you are constantly working with a virtualenv you can also put the
 bottom of the file.  That way every time you activate your virtualenv you
 automatically also activate the correct application name.
 
+You need to edit the specific activate script for different shell, for example:
```

Comment:
There are different scripts depending on the shell you're using. This sentence is awkward though, should read "Edit the activate script for the shell you use. For example:"